modules.d/06rngd/module-setup.sh

```diff
@@ -32,7 +32,12 @@ depends() {
 
 install() {
     inst rngd
-    inst_simple "${moddir}/rngd.service" "${systemdsystemunitdir}/rngd.service"
+    inst_simple "${systemdsystemunitdir}/rngd.service"
```

Comment:
This would mean we'd still ship the service file. Why do that, esp. in case there's no such service file on the system - I think dracut shouldn't be the delivery mechanism for such configuration.

Or have misunderstood the issue?

Reply:
Great! I think even originally, dracut was just keeping a copy of something that was shipped by default everywhere, because it needed to be modified (which is currently done with the `RNGD_ARGS`).